.github/workflows/test.yml

```diff
@@ -74,6 +77,9 @@ jobs:
       - name: Run deployment
         run: |
           ./foremanctl deploy --certificate-source=${{ matrix.certificate_source }}
+      - name: Setup hammer
+        run: |
+          ./foremanctl setup-hammer
```

Comment:
no `--certificate-source`, as it's supposed to remember that from the step before!